SUPPORT.md

```diff
@@ -0,0 +1,6 @@
+# Support
+
+Thank you for using Metriport! If you are experiencing problems or have questions, the following are the preferred methods of reaching us:
+
+1. Report a new issue using the [issue tracker](https://github.com/metriport/metriport/issues).
+2. If you're a Metriport customer, reach out on Slack or [contact us via our website](https://www.metriport.com/contact-us).
```

Comment:
perhaps we can just give them our contact@ email here for now?

we have everyone on Slack now since it's the early days, but might not want to bind ourselves to doing this for *everyone* in the future

Reply:
Sounds good.